src/index_runner/indexers/main.py

```diff
@@ -49,6 +49,13 @@ def index_obj(msg_data, es_queue):
     except WorkspaceResponseError as err:
         print('Workspace response error:', err.resp_data)
         raise err
+
+    # check if this particular object has the tag "noindex"
+    metadata = ws_info[-1]
+    if metadata.get('searchtags'):
```

Comment:
can you just change to `metadata.get('searchtags') and 'noindex' in metadata['searchtags']:` so we can get rid of one indentation level

Reply:
Nevermind it really doesn't matter